specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```
@@ -0,0 +1,1443 @@
+        "operationId": "MachineLearningInferenceEndpointDeployment_Update",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update Deployment": {
+            "$ref": "./examples/updateDeployment.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointDeploymentNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "properties",
+            "description": "The payload that is used to update the Inference Deployment.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointDeploymentUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Inference Deployment update was successful.",
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointDeploymentResource"
+            }
+          },
+          "202": {
+            "description": "Inference Deployment update initiated.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URI to poll for asynchronous operation status.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferenceEndpoints/{inferenceEndpointName}/deployments/{deploymentName}/skus": {
+      "get": {
+        "tags": [
+          "MachineLearningInferenceEndpointDeployments"
+        ],
+        "x-ms-examples": {
+          "Get Sku setting info for deployment": {
+            "$ref": "./examples/getDeploymentSkuInfo.json"
+          }
+        },
+        "summary": "This is for an ARM Contract with Azure Monitor to scale the number of instances of the Inference Deployment.",
+        "description": "Get SKU information for scaling an Inference Deployment.",
+        "operationId": "MachineLearningInferenceEndpointDeployment_ListDeploymentSkus",
```

Align with the operationId naming convention
```suggestion
        "operationId": "MachineLearningInferenceEndpointDeployments_ListDeploymentSkus",
```